keyboards/exclusive/e6_rgb/e6_rgb.c

```diff
@@ -190,3 +176,21 @@ led_config_t g_led_config = { {
 } };
 
 #endif
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    switch(keycode) {
+#if defined(RGBLIGHT_ENABLE) && defined(RGB_MATRIX_ENABLE)
+        case KC_F13: // toggle rgb matrix
+        rgb_matrix_toggle();
```

Comment:
I'd prefer some indents in the body of each case statement

Reply:
add indents :)